purchase_order_hide_receipt_status/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module hides receipt_status field from purchase order form view and tree view to
+prevent user confusion with reception_status field when purchase_reception_status module
+installed.
```

Comment:
Fix the grammar error and refine this description a bit.

Reply:
update the description on the new commit. thanks Tashiro san!